pkgs/sops-install-secrets/main.go

```diff
@@ -870,6 +884,21 @@ func installSecrets(args []string) error {
 		return err
 	}
 
+	if manifest.UserMode {
+		rundir, ok := os.LookupEnv("XDG_RUNTIME_DIR")
+		if !ok {
+			rundir = fmt.Sprintf("/run/user/%d", os.Getuid())
```

Comment:
if `XDG_RUNTIME_DIR` is not set, it's likely that there is no `/run/user/<uid>`. I would rather return an error here rather than writing to some random directory.